<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TalantOO</class>
 <widget class="QDialog" name="TalantOO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>611</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="offlinePbt">
        <property name="text">
         <string>离线消息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sendmessLet">
        <property name="placeholderText">
         <string>在此输入消息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendmessPbt">
        <property name="text">
         <string>发送消息</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>目标用户</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usrLet">
        <property name="placeholderText">
         <string>在此输入连接用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connectPbt">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitPbt">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="sendfilePbt">
      <property name="text">
       <string>文件传输</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
